clusterloader/config/rctest.yaml

```diff
@@ -0,0 +1,38 @@
+provider: local
```

Comment:
e2e wants a provider, it has some other fn's in `e2e.go` that are dependent on it afaik. I can remove this.